### ISSUE TITLE
Test failing in codespaces-linux - php not in path

### ISSUE THREAD

**Issue by Chuxel** [microsoft/vscode-dev-containers#322]
Since updating to the latest oryx image, the `codespaces-linux` image no longer appears to have PHP in the path.  We should either remove it from `test.sh` or correct that.

```text
codespace:~/workspace$ test-project/test.sh 

🧪  Testing vscode-server: [ 1 -ge 1 ]
🏆  Passed!

🧪  Testing non-root-user: id codespace
uid=1000(codespace) gid=1000(codespace) groups=1000(codespace),800(docker)
🏆  Passed!

🧪  Testing /home/codespace: [ -d /home/codespace ]
🏆  Passed!

🧪  Testing sudo: sudo -u codespace echo sudo works.
sudo works.
🏆  Passed!

🧪  Testing git: git --version
git version 2.11.0
🏆  Passed!

🧪  Testing command-line-tools: which top ip lsb_release
/usr/bin/top
/sbin/ip
/usr/bin/lsb_release
🏆  Passed!

🧪  Testing oryx: oryx platforms
Platform  : dotnet
Versions  : 1.0.16
            1.1.13
            2.0.9
            2.1.17
            2.2.7
            3.0.3
            3.1.3
Properties: project - Relative path of the project file to build.

Platform  : nodejs
Versions  : 4.4.7
            4.5.0
            4.8.0
            6.2.2
            6.6.0
            6.9.3
            6.10.3
            6.11.0
            6.17.1
            8.0.0
            8.1.4
            8.2.1
            8.8.1
            8.9.4
            8.11.2
            8.12.0
            8.15.1
            8.17.0
            9.4.0
            10.1.0
            10.10.0
            10.14.2
            10.16.3
            10.19.0
            12.16.1
Properties: npm_registry_url - Custom npm registry URL. Will be written to .npmrc during the build.
            compress_node_modules - Indicates how and if 'node_modules' folder should be compressed into a single file in the output folder. Options are 'zip', and 'tar-gz'. Default is to not compress. If this option is used, when running the app the node_modules folder must be extracted from this file.
            prune_dev_dependencies - When using different intermediate and output folders, only the prod dependencies are copied to the output. Options are 'true', blank (same meaning as 'true'), and 'false'. Default is false.
            require_build - Requires either 'npm run build' or 'yarn run build' or custom run build command  to be run. Default is false. If value is not provided, it is assumed to be 'true'.

Platform: php
Versions: 5.6.40
          7.0.33
          7.2.28
          7.3.15
          7.4.3

Platform  : python
Versions  : 2.7.17
            3.6.10
            3.7.7
            3.8.2
Properties: virtualenv_name - Name of the virtual environment to be created. Defaults to 'pythonenv<Python version>'.
            compress_virtualenv - Indicates how and if virtual environment folder should be compressed into a single file in the output folder. Options are 'zip', and 'tar-gz'. Default is to not compress. If this option is used, when running the app the virtual environment folder must be extracted from this file.
            packagedir - If provided, packages will be downloaded to the given directory instead of to a virtual environment.


🏆  Passed!

🧪  Testing python: python --version
Python 2.7.13
🏆  Passed!

🧪  Testing dotnet: dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.1.201
 Commit:    b1768b4ae7

Runtime Environment:
 OS Name:     debian
 OS Version:  9
 OS Platform: Linux
 RID:         debian.9-x64
 Base Path:   /opt/dotnet/sdks/3.1.201/sdk/3.1.201/

Host (useful for support):
  Version: 3.1.3
  Commit:  4a9f85e9f8

.NET Core SDKs installed:
  1.1.14 [/home/codespace/.dotnet/sdk]
  2.1.805 [/home/codespace/.dotnet/sdk]
  2.2.402 [/home/codespace/.dotnet/sdk]
  3.0.103 [/home/codespace/.dotnet/sdk]
  3.1.201 [/home/codespace/.dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.17 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.7 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.17 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.7 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.3 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.3 [/home/codespace/.dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 1.1.13 [/home/codespace/.dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.17 [/home/codespace/.dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [/home/codespace/.dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.3 [/home/codespace/.dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.3 [/home/codespace/.dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
🏆  Passed!

🧪  Testing node: node --version
v12.16.1
🏆  Passed!

🧪  Testing php: php --version
test-project/test.sh: line 15: php: command not found
💥  php check failed.

🧪  Testing bash: bash --version
GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
🏆  Passed!

🧪  Testing fish: fish --version
fish, version 2.4.0
🏆  Passed!

🧪  Testing zsh: zsh --version
zsh 5.3.1 (x86_64-debian-linux-gnu)
🏆  Passed!

💥  Failed tests: php
```

**Comment by Chuxel** [comment]
(Also, I assume php should be available in the path, so this is not a test bug.)

**Comment by Chuxel** [comment]
Resolved in #362
